InscryptionAPI/Card/CardExtensions.cs

```diff
@@ -1119,7 +1121,7 @@ public static bool DoesNotHaveSpecialAbility(this CardInfo cardInfo, SpecialTrig
     /// <returns>true if the ability does not exist</returns>
     public static bool DoesNotHaveAbility(this PlayableCard playableCard, Ability ability)
     {
-        return !playableCard.HasAbility(ability);
+        return playableCard.Info.DoesNotHaveAbility(ability);
```

Comment:
This needs to account for PlayableCard.TemporaryMods, not just the abilities on PlayableCard.Info